src/pages/policies/BackupPolicy.tsx

```diff
@@ -167,19 +167,26 @@ export const BackupPolicy = ({type}) => {
   const possibleAddRule = () =>
     model.rules.some(
       ({
-         ruleBackupDestinations: {
-           primaryBackupDestination: {backupDestination: pbd},
-           secondaryBackupDestination: {backupDestination: sbd} = {
-             backupDestination: null,
-           },
-         },
-       }) => !!pbd === !!pbd?.guid && !!sbd === !!sbd?.guid,
+        ruleBackupDestinations: {
+          primaryBackupDestination: { backupDestination: pbd },
+          secondaryBackupDestination: { backupDestination: sbd } = {
+            backupDestination: null,
+          },
+        },
+      }) => !!pbd === !!pbd?.guid && !!sbd === !!sbd?.guid,
     );
 
   useEffect(() => {
     handleChangeBackupDestination();
   }, [model.rules]);
 
+  const itemVWTemplate = (option) => (
+    <div>
+      <p>{option.name}</p>
+      <p style={{ fontSize: '0.8em', color: 'grey' }}>{option.guid}</p>
```

Comment:
You can make it look better by destructuring the **option**: ** itemVWTemplate = ({name, guid}) => {**